cmd/kubelet/app/server.go

```diff
@@ -1020,3 +1027,11 @@ func parseResourceList(m utilconfig.ConfigurationMap) (api.ResourceList, error)
 	}
 	return rl, nil
 }
+
+func parseKernelTunableBehaviour(protect bool) cm.KernelTunableBehavior {
```

Comment:
The problem is that kubelet is consumed in different methods. One if via the cmd/kubelet package. It also gets linked into other binaries. I'd prefer not supporting the `warn` option at all for now for the same reasons mentioned above. Changing internal code is much simpler in the future than the CLI, so we can switch to a string easily in the future.